fineract-provider/build.gradle

```diff
@@ -47,7 +47,7 @@ buildscript {
         classpath 'org.zeroturnaround:gradle-jrebel-plugin:1.1.10'
         classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.3.1.RELEASE'
         classpath 'gradle.plugin.com.hierynomus.gradle.plugins:license-gradle-plugin:0.15.0'
-        classpath 'org.apache.openjpa:openjpa:3.1.1' // when upgrading, also change OpenJPA version repeated below in dependencyManagement!
+        classpath 'org.apache.openjpa:openjpa:3.1.2' // when upgrading, also change OpenJPA version repeated below in dependencyManagement!
```

Comment:
@awasum did you see the inline comment here? I wrote this hoping so that when we get PRs like this, someone would remember to also do #1180 - otherwise they diverge.